dashboards-observability/.cypress/integration/1_event_analytics.spec.js

```diff
@@ -787,6 +863,249 @@ describe('Renders chart and verify Toast message if X-axis and Y-axis values are
+    cy.get('.euiFieldText.euiColorPicker__input.euiFieldText--withIcon').click();
+    cy.get('[aria-label="Select #D36086 as the color"]').click();
+    cy.get('.euiButton__text').contains('Preview').click();
+    cy.get('.trace.treemap path[style*="rgb(211, 96, 134)"]').should('exist');
+  });
+
+  it('Parent field not available under color theme', () => {
+    cy.get('.euiTitle.euiTitle--xxsmall').contains('Color Theme').should('exist');
+    cy.get('.euiSuperSelectControl').contains('Default').click();
+    cy.get('.euiContextMenuItem__text .euiColorPalettePicker__item').eq(1).contains('Single color').click();
+    cy.get('.euiFieldText.euiColorPicker__input.euiFieldText--withIcon').click();
+    cy.get('[aria-label="Select #54B399 as the color"]').should('exist').click();
+    cy.get('.euiButton__text').contains('Preview').click();
+    cy.get('.euiSuperSelectControl').click();
+    cy.get('.euiContextMenuItem__text .euiColorPalettePicker__item').eq(2).contains('Multicolored').click();
+    cy.wait(delay);
+    cy.get('.euiFormHelpText.euiFormRow__text').eq(1).contains('Child field').should('exist');
+    cy.get('.euiFieldText.euiColorPicker__input.euiFieldText--withIcon').click();
+    cy.get('[aria-label="Select #D36086 as the color"]').click();
+    cy.get('.euiFormHelpText.euiFormRow__text').eq(1).contains('Parent field').should('not.exist');
+    cy.get('.euiButton__text').contains('Preview').click();
+    cy.get('.trace.treemap path[style*="rgb(211, 96, 134)"]').should('exist');
+  });
+
+  it('Render Tree Map chart and verify Data Configuration panel', () => {
+    renderDataConfigTreeMap();
+    cy.get('.trace.treemap').should('exist');
+  });
+
+  it('Renders Tree Map and Add Multiple Parent', () => {
+    cy.get('.euiResizablePanel.euiResizablePanel--middle').contains('Data Configurations');
+    cy.get('.euiTitle.euiTitle--xxsmall').eq(1).contains('Dimensions');
+    cy.get('.first-division .euiFormLabel.euiFormRow__label').eq(0).contains('Child Field');
+    cy.get('.euiComboBoxPill.euiComboBoxPill--plainText').eq(0).click({force: true});
+    cy.get('.euiComboBoxOption__content').eq(3).click();
+    cy.get(' [data-test-subj="addParentButton"] .euiButton__text').contains('+ Add Parent').click();
+    cy.get('.first-division .euiFormLabel.euiFormRow__label').contains('Parent 1').should('exist');
+    cy.get('p.euiComboBoxPlaceholder').eq(0).click({ force: true });
+    cy.get('.euiComboBoxOption__content').eq(0).click();
+    cy.get(' [data-test-subj="addParentButton"] .euiButton__text').contains('+ Add Parent').click();
+    cy.get('.first-division .euiFormLabel.euiFormRow__label').contains('Parent 2').should('exist');
+    cy.get('p.euiComboBoxPlaceholder').click({ force: true });
+    cy.get('.euiComboBoxOption__content').eq(1).click();
+    cy.get(' [data-test-subj="addParentButton"] .euiButton__text').contains('+ Add Parent').click();
```

Comment:
Adding parent steps can be moved to common method.

Reply:
ok will make a common method